pages/guides/how-to-schedule-price-updates-with-gelato.mdx

```diff
@@ -0,0 +1,125 @@
+# How to Schedule Price Updates with Gelato
+
+This guide explains how to create an automated task that updates Pyth price feeds on a schedule
+
+### Why Gelato? Introduction to Gelato
+
+Gelato is web3’s decentralized backend empowering builders to create augmented smart contracts that are automated,
```

Comment:
maybe link to the gelato homepage?

or a link to documentation for further reading?

Reply:
also the word "builders" is a pet peeve of mine. The right word is "developers"